articles/search/vector-search-how-to-generate-embeddings.md

```diff
@@ -69,7 +69,12 @@ print(embeddings)
 <!-- 
 + We've done proof-of-concept testing of embeddings for a thousand images using [image retrieval vectorization in Cognitive Services](/azure/cognitive-services/computer-vision/how-to/image-retrieval). We hope to provide a demo of this soon. -->
 
-+ Similarity search expands your options for searchable content, for example by matching image content with text content, or matching across multiple languages. But not every query is improved with vector search. Keyword matching with BM25 is cheaper, faster, and easier, so integrate vector search only where it adds value.
++ **Identify use cases:** Evaluate the specific use cases where embedding model integration for vector search features can add value to your search solution. This can include matching image content with text content, cross-lingual searches, or finding similar documents.
++ **Optimize cost and performance**: Vector search can be resource-intensive, so consider only vectorizing the fields that contain semantic meaning
```

Comment:
> Vector search can be resource-intensive and highly resource limited, so consider only vectorizing the fields that contain semantic meaning

Reply:
See https://github.com/MicrosoftDocs/azure-docs-pr/pull/244304  Porting a slightly modified version (highly resource limited sounds a little harsh?):  "Vector search can be resource-intensive and is subject to maximum limits, so consider only vectorizing the fields that contain semantic meaning

Reply:
I like that, thanks for the edit!